docs/self-hosted/docker-guide.md

```diff
@@ -153,14 +141,13 @@ increment the tag version number, e.g.:
 You can then issue the following two commands (from your docker-compose root):
 
 ```bash
-docker-compose pull
-docker-compose up -d
+docker compose pull
```

Comment:
If you use versioned images in your compose file, I believe `docker compose up` automatically pulls if the image doesn't exist yet 🤔

Reply:
Correct 👍